client/a8c-for-agencies/sections/purchases/controller.tsx

```diff
@@ -91,6 +105,10 @@ export const paymentMethodsContext: Callback = ( context, next ) => {
 };
 
 export const paymentMethodsAddContext: Callback = ( context, next ) => {
+	if ( isEnabled( 'a4a-bd-checkout' ) ) {
+		page.redirect( EXTERNAL_WPCOM_PAYMENT_METHODS_URL );
```

Comment:
Is there a reason this one uses `page.redirect()` while the others use `window.location.href =`?

Reply:
fixed